app/javascript/modules/listingDetailsLottery/ListingDetailsLotteryResultsRow.scss

```diff
@@ -1,4 +1,5 @@
 .lottery-results-row {
+  align-items: center;
```

Comment:
question: did you confirm this styling change with designers? I can't find any mention of it.

Reply:
I did not! Without this, now the the block of text is bigger, it will look like: 
![Screenshot 2024-04-19 at 9 40 32 AM](https://github.com/SFDigitalServices/sf-dahlia-web/assets/43218114/ab40a878-58a3-454f-9a7d-2eea50609d2e)
which I assumed is something Yindi would flag - I'll comment on the ticket to ask

Reply:
yindi said top aligned is fine, so I'll remove the scss